application/controllers/template_controller.py

```diff
@@ -31,18 +31,7 @@ def validate_template(template):
         raise InvalidTemplateException(exception.message, status_code=400)
 
 
-def upload_comms_template(template_id, template_object):
-
-    logger.info("Uploading comms template with id", id=template_id)
-
-    validate_template(template_object)
-
-    existing_template = get_template_by_id(template_id)
-
-    if existing_template:
-        logger.info("Attempted to upload already existing template", id=template_id)
-        raise InvalidTemplateException(PREEXISTING_TEMPLATE, status_code=400)
-
+def send_template_to_db(template_id, template_object):
```

Comment:
Probably don't need to_db again. Might be worth being explicit in that it will create_or_update_template?